File: ms_cinema/applications/app-service/src/main/java/co/com/luisgomez29/config/secretsmanager/SecretsManagerProperties.java
package co.com.luisgomez29.config.secretsmanager;

import org.springframework.boot.context.properties.ConfigurationProperties;

@ConfigurationProperties(prefix = "helpers.secrets-manager")
public record SecretsManagerProperties(
        String endpoint,
        Integer cacheSize,
        Integer cacheTime,
        String postgres
) {
}

File: ms_cinema_kt/applications/app-service/src/main/kotlin/co/com/luisgomez29/config/secretsmanager/SecretsManagerProperties.kt
package co.com.luisgomez29.config.secretsmanager

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties(prefix = "helpers.secrets-manager")
data class SecretsManagerProperties(
    val endpoint: String,
    val cacheSize: Int,
    val cacheTime: Int,
    val postgres: String
)